spec.html

```diff
@@ -4264,6 +4264,9 @@ <h1>
             1. [id="step-getvalue-toobject"] Let _baseObj_ be ? ToObject(_V_.[[Base]]).
             1. If IsPrivateReference(_V_) is *true*, then
               1. Return ? PrivateGet(_baseObj_, _V_.[[ReferencedName]]).
+            1. If _V_.[[ReferencedName]] is not a property key, then
+              1. Let _referencedName_ be ? ToPropertyKey(_V_.[[ReferencedName]]).
+              1. Set _V_.[[ReferencedName]] to _referencedName_.
```

Comment:
Here and elsewhere, I'd just collapse these two steps into one.

Reply:
Gladly; for some reason I thought this was preferred.